test/types/api-responses.ts

```
@@ -0,0 +1,114 @@
+  error?: string;
+}
+
+export interface SheetSetupProgressData {
+  setupId: string;
+  status: string;
+  progress: SetupProgress;
+}
+
+export type AuthStartResponse = BaseSuccessResponseWithData<AuthStartData>;
+export type AuthCallbackResponse = BaseSuccessResponseWithData<AuthCallbackData>;
+export type AuthLogoutResponse = BaseSuccessResponseWithData<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)